packages/typespec-test/test/hierarchy_generic/generated/typespec-ts/package.json

```diff
@@ -70,6 +69,7 @@
     "vitest": "^3.0.3",
     "@azure-tools/test-credential": "^2.0.0",
     "@azure-tools/test-recorder": "^4.0.0",
+    "@microsoft/api-extractor": "^7.40.3",
```

Comment:
I don't think we are leveraging dev tool in non azure generation?

Reply:
That's right. We need this dependency and the script below runs `api-extractor run --local`.  Only azure mono repo uses dev-tool and won't need this dependency.